### PR TITLE
Add kiwi-image-tbz-requires to provide kiwi-image:tbz

### PR DESCRIPTION
**Pull request by andreas-schwab**

### CODE REVIEW

File: custom_boot/package/kiwi-boot-descriptions.spec [hunk 1]
```diff
@@ -192,6 +192,16 @@ Requires:       jing
 Meta package for the buildservice to pull in all required packages
 for the build host to build simple disk images
 
+%package -n kiwi-image-tbz-requires
+Summary:        KIWI - buildservice host requirements for root archive tarball
+Group:          System/Management
+Provides:       kiwi-image:tbz
+Requires:       kiwi-filesystem-requires
```
Comment (schaefi) [inline custom_boot/package/kiwi-boot-descriptions.spec]: why do we need filesystem-requires for creating an archive of the root tree ? I thought a tar requirement should be enough ?

**Review by schaefi — COMMENTED** [review]
Looks good, just the question why filesystem tools needs to be required. Thanks

File: custom_boot/package/kiwi-boot-descriptions.spec (hunk 1 above)
Comment (andreas-schwab) [reply custom_boot/package/kiwi-boot-descriptions.spec]: I have just copied from the other kiwi-image:* packages.  Probably tar + xz should be enough.

**Comment by schaefi** [comment]
:+1:

**Pull request merged**